modmesh/app/euler1d.py

```diff
@@ -529,11 +553,31 @@ def start(self):
     def stop(self):
         self.timer.stop()
 
+    def step_button_cb(self):
```

Comment:
According to https://wiki.qt.io/How_to_Use_QPushButton
`QPushButton` clicked callback function has a boolean argument `checked`, if I bind `self.step()` directly, then `step()` get a 0 step marching, that is not expected behavior of the step button.

Reply:
Then why this function does not need a Boolean argument?

I am fine having the function to take no arguments, as long as it works.  Just curious about the reason.